__tests__/app.test.js

```diff
@@ -157,3 +157,67 @@ describe('GET /api/articles', () =>
         })
     })
 })
+
+})
+
+describe('GET: /api/articles/:article_id/comments', () =>
+    {
+        test('Responds with an array of objects', () =>
+        {
+            return request(app).get('/api/articles/1/comments').expect(200)
+            .then(({ body }) =>
+            {
+                expect(Array.isArray(body)).toBe(true)
+                body.forEach((element) =>
+                {
+                    expect(typeof element).toBe('object')
+                    expect(Array.isArray(element)).toBe(false)
+                })
+            })
+        })
+        test(`Responds with an array of objects that have the following properties: 
+        comment_id, votes, created_at, author, body, article_id`, () =>
+        {
+            return request(app).get('/api/articles/1/comments').expect(200)
+            .then(({ body }) =>
+            {
+                body.forEach((element) =>
+                {
+                    expect(element).toHaveProperty('comment_id')
+                    expect(element).toHaveProperty('votes')
+                    expect(element).toHaveProperty('created_at')
+                    expect(element).toHaveProperty('author')
+                    expect(element).toHaveProperty('body')
+                    expect(element).toHaveProperty('article_id')
+                })
+            })
+        })
+        test('responds with the correct comments of the relevant article', () =>
+        {
+            return request(app).get('/api/articles/1/comments').expect(200)
+            .then(({ body }) =>
+            {
+                body.forEach((element) =>
```

Comment:
Make sure we check for the amount of comments before using forEach in our tests. If there were a case where there was nothing to iterate over, we would never execute the forEach( )  and could leave ourselves open to false positives in our tests.